.claude/scripts/board-claim-token-contract.test.sh

```
@@ -0,0 +1,51 @@
+grep -Fq 'board-claim:<lane>' "${constitution}" ||
+  fail "constitution does not require the greppable board-claim:<lane> token"
+grep -Fq 'exactly `claude`, `codex`, or `cursor`' "${constitution}" ||
+  fail "constitution does not pin the three live lanes as the only admitted board-claim values"
+grep -Fq "timed from the comment's \`created_at\`" "${constitution}" ||
+  fail "constitution does not time the board-only lease from the claim comment's created_at"
+grep -Fq 'Reply to your own claim comment when finished' "${constitution}" ||
+  fail "constitution does not require reply-to-close on a board-only claim"
+
+grep -Fq 'board-claim:<lane>' "${run_loop}" ||
+  fail "portfolio-maintenance board Act step does not require board-claim:<lane>"
+grep -Fq '`board-claim:claude`' "${run_loop}" ||
```

Shellcheck (0.11.0)

[info] Expressions don't expand in single quotes, use double quotes for that.

(SC2016)